examples/wallet/app/reducers/account.js

```diff
@@ -31,6 +32,11 @@ export default function account(
     return { transactions: [] };
   }
   switch (action.type) {
+    case SET_VALID_UNLOCK_WALLET_PASSWORD:
```

Comment:
when you add a reducer case for an action, you also need to add the type of the action, otherwise the type system'll warn you that it's impossible to get to the new case.

Reply:
This reducer is not used anymore